src/main/java/transforms/DateTimeDiffTransform.java

```diff
@@ -45,7 +43,7 @@ public DateTimeDiffTransform(
 
     @Override
     public Object apply( Map<String, String> row ) {
-        if ( !( row.containsKey( columns.get(0) ) || row.containsKey( columns.get(1) ) ) ) {
+        if ( !( row.containsKey( columns.get( 0 ) ) || row.containsKey( columns.get( 1 ) ) ) ) {
```

Comment:
I find it way easier to read `if ( !a && !b )` than `if ( !( a || b) )` so I'd vote to change this up to that.

Also, are columns 0 and 1 the ones that contain dates to diff? Will there always be only 2? If so we should add a check verifying the lengths of `columns` is 2 (although I'd prefer breaking it up into 2 parameters, like `dateTimeCol1` and `dateTimeCol2`)

Reply:
Added in a check.  Not gonna break up into two parameters since this will break ongoing integrations.